### DISCUSSION TITLE
Possible to add a second abrp to a single psa_controller?

### DISCUSSION THREAD

**Discussion by tkoeberl** [flobz/psa_car_controller#695] [Q&A]
Hi,

I already connected my own abrp account to psa_car_controller. Is it possible to add a second abrp account?

thanks,
-thomas

**Comment by Slin-nilS** [comment]
Is sharing the car in ABRP vehicle config an idea? 
![image](https://github.com/flobz/psa_car_controller/assets/95978369/dea11791-7537-428c-9c8c-cb17e8f6b9b8)

**Comment by tkoeberl** [comment]
Only possible with Premium.  😕

**Comment by harrymr740** [comment]
great info!
